sui/types-utils.js

```diff
@@ -83,12 +83,18 @@ const discoveryStruct = bcs.struct('Discovery', {
     fields: discoveryTable,
 });
 
-const tableStruct = bcs.struct('Table', {
+const bagStruct = bcs.struct('Bag', {
     id: UID,
     size: bcs.U64,
 });
 
-const bagStruct = bcs.struct('Bag', {
+const operatorsStruct = bcs.struct('Operators', {
```

Comment:
this should be defined in cgp-sui? And we should remove duplicates here and use the cgp-sui ones

Reply:
done https://github.com/axelarnetwork/axelar-cgp-sui/pull/105/files